config/locales/en/errors.yml

```diff
@@ -111,3 +111,11 @@ en:
           attributes:
             add_offence:
               inclusion: Select yes if you want to add another offence
+        steps/case/offence_date_fieldset_form:
+          attributes:
+            date:
+              future_not_allowed: Offence date cannot be in the future
+          summary:
+            date:
+              blank: Offence dates cannot be blank
```

Comment:
I believe it will still worth to have indexed summary errors otherwise if more than one date fails validation there will be repetition of messages in the summary and is something we want to avoid for screen readers.
Even if we don't show in the page "Offence date 1", "Offence date 2"... we will have that as visually hidden in each of the dates and thus it makes sense, but we can discuss along with the simplification of the mark-up 👍

Reply:
If we do that it would be just like this: `blank: Enter offence date %{index}` because the `index` is already exposed.
I suppose is a matter of carefully crafting proper error messages if we do that. Probably one for Carolina.

It can be done as a separate PR anyways.